server_code/historic_server/persistence.py

```diff
@@ -0,0 +1,204 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+from uuid import uuid4
+
+from anvil.tables import app_tables, in_transaction, order_by
+
+from anvil_extras.server_utils import LOGGER
```

Comment:
We should avoid depending on `anvil_extras` if another user depends on a different version this will create a dependency conflict.

Reply:
That dependency only needs to exist whilst the module is in anvil labs. Once it's in extras, it's not necessary. Does that help?

Reply:
maybe we can just have a rule that if you depend on anvil-labs you have to depend on the X version of anvil-extras.
Where X is probably the published version